images/kcp-registrar/register.sh

```diff
@@ -20,54 +20,58 @@ set -o pipefail
 
 usage() {
 
-    printf "Usage: KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" DATA_DIR="/workspace" ./register.sh\n\n"
+    printf "Usage: KCP_ORG="root:pipeline-svc" KCP_WORKSPACE="compute" DATA_DIR="/workspace" ./register.sh\n\n"
 
     # Parameters
     printf "The following parameters need to be passed to the script:\n"
-    printf "KCP_ORG: the organistation for which the workload clusters need to be registered, i.e.: root:pipelines-service\n"
+    printf "KCP_ORG: the organization for which the workload clusters need to be registered, i.e.: root:pipeline-svc\n"
     printf "KCP_WORKSPACE: the name of the workspace where the workload clusters get registered (created if it does not exist), i.e: compute\n"
     printf "DATA_DIR: the location of the cluster files\n"
     printf "          a single file with extension kubeconfig is expected in the subdirectory: credentials/kubeconfig/kcp\n"
     printf "          kubeconfig files for compute clusters are expected in the subdirectory: credentials/kubeconfig/compute \n\n"
 }
 
-prechecks () {
+exit_error() {
+    printf "[ERROR] %s\n\n" "$@" >&2
+    usage
+    exit 1
+}
+
```

Comment:
I don't think this modification is worth the time for review and validation. The gain is negligible at best and it enforces a formatting that may not always be welcome.